Add VaProcessList to ds-playground page

nickjg231 opened the pull request:
## Summary

- Add the V1 and V3 `<VaProcessList>` web-component to the DS playground page

## Related issue(s)
https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1966

## Testing done
- Manual testing

## Screenshots

Desktop:
![image](https://github.com/department-of-veterans-affairs/vets-website/assets/15200011/9a9fdfdd-116e-4393-8d01-9ec2e41e1b96)

V1 Mobile:
![image](https://github.com/department-of-veterans-affairs/vets-website/assets/15200011/eb6580c5-3fce-42e4-8590-0fe7d40a27b7)

V3 Mobile:
![image](https://github.com/department-of-veterans-affairs/vets-website/assets/15200011/0914650f-bbc9-4546-ad71-f3cab9d26436)



## What areas of the site does it impact?

- The DS playground page

## Acceptance criteria

- [ ] The DS playground page has a section for the VaProcessList web-component
- [ ] The section has both a V1 and V3 version of the component

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

OptionSelect committed:
61700 - [FSR] - Review Display - Vehicles (#24896)

* Add vehicles readonly review

* Update styling per Andrew

* Add year ternary

* Fix year ternary thanks to Andrew!

mmoyer-va committed:
[#MHV-45445] Flatten side navigation and remove health history link (#24903)

* [#MHV-45445] Flatten side navigation and remove health history link

* [#MHV-45445] Removed health-history check from Cypress -- page no longer exists

gerard-sullivan committed:
MHV-46792-update (#24866)

* MHV-46792-update

* MHV-46792-update

jsimonVA committed:
#VFEP-709 #comment ageWarning and Highschool question trigger update (#24888)

rmessina1010 committed:
address protobufjs vulnerability (copy) (#24904)

* patches protobufjs upto 6.11.3

* upgrade dd-trace to 2.24.0 to fix secondary protobfjs vulnerability

* spaaces around vets-json-schema

* sets dd-trace version to absolute, as a test

mmoyer-va committed:
MHV-45447 Split Share and Download page into two pages (#24908)

* [#MHV-45447] Split Share & Download into two pages

* [#MHV-45447] Fixed linting errors and added aria tags

fazilqa committed:
upd secure-messaging-folders.cypress.spec.js (#24911)

Thrillberg committed:
Modify submit-transformer for 21-0972 to handle booleans (#24905)

* Modify submit-transformer for 21-0972 to handle booleans

* Fix lint and remove mock submit

* Fix import path and refactor function

mattwrightva committed:
MHV-42360: Labs and tests, microbiology pdf generation complete (#24915)

Wyzardsleeves committed:
MHV-42364 Added PDF download for EKG (#24898)

* added PDF download for EKG

* removed 'list' from the download button

OptionSelect committed:
61704 - [FSR] - Review Display - Installment Contracts (#24900)

* Add installment contracts readonly review section

* Fix original amount display bug

* Fix duplicate keys and unnecessary space

* Fix double ampersand render potential issues

alex-morgun-1 committed:
[MHV-46068] SM to VA.gov - print modal radio label (#24890)

* [MHV-46068] print modal radio label

* [MHV-46068] print modal unit test coverage

* [MHV-46068] add description to radio label

fazilqa committed:
Update secure-messaging-draft-save-with-attachments.cypress.spec.js (#24919)

* Update secure-messaging-draft-save-with-attachments.cypress.spec.js

* Update secure-messaging-folders.cypress.spec.js

---------

Co-authored-by: haniasma <121971510+haniasma@users.noreply.github.com>

mmoyer-va committed:
MHV-45446 Merge Health History page into permanent Landing Page (#24921)

* [#MHV-45446] Merge Health History page into permanent Landing Page

* [#MHV-45446] Added va-introtext class

* [#MHV-45446] Removed Health History unit tests

ataker committed:
Add direct import of i18n-setup (#24883)

* Add direct import of i18n-setup

* move import

* ignore deprecated component

will be properly removed in subsequent PR

acrollet committed:
Update CODEOWNERS for DLC supply re-ordering application (#24927)

The health apps team is taking over maintenance and feature additions for this application.

batemapf committed:
DLC supply reorder - add basic api mocks (#24907)

* add basic api mocks

* Update feature toggle name to match vets-api PR

---------

Co-authored-by: Adrian Rollett <adrian.rollett@va.gov>

ataker committed:
replace react component with web component and update tests (#24929)

Commit:
Notification Settings - Add Email as channel and consume default send indicator (#24848)

* wip: supporting email

* feat: filter unavailable items

* feat: MissingContact expandable, hook creation

* feat: consume default send, filter unavail with toggles

* test: unit test notif setting hooks pt 1

* test: update unit tests for hook usage

* fix: clean up notif settings display logic, update e2e tests

* fix: remove dup tests, spelling, address feedback

* fix: clean up and standardize, thnx allison

longmd committed:
Increase unit test coverage for healthcare application (#24916)

jamigibbs committed:
Update design system component library to latest version v17.0.1 (#24924)

* Update component library to latest version

* Move placement of back to top component

* Update test

* Adjust wrapping div for back to top

OptionSelect committed:
61705 - [FSR] - Review Display - Employment Questions (#24920)

* Add installment contracts readonly review section

* Fix original amount display bug

* Fix duplicate keys and unnecessary space

* Fix double ampersand render potential issues

* Add employment questions to review page

* Update proptypes

Commit:
fix: loading indicator logic, add test (#24934)

allisonlu committed:
Notification Center MVP - create mock server and data (#24735)

* add application scaffolding

* fix codeclimate error

* - set up mock server with example data

* set up actions and reducers for Notifications

* added Notification component

* cleanup and reorganizing

* clean up:
- removed anything to do w dismissing
- remove irrelevant feature toggle

* cleanup: remove duplicate code

* moved files in /notification-center to
/personalization/notification-center

* created reused actions, mocks, and component for
notifications in common folder within dashboard

* - removed redundant files
- updated paths to common files

* move notification reducers to common folder

* - changed paths in /dashboard/ application to use
common notificatoin actions, reducers, mocks, and components
- removed redundant files

* - renamed notification component that was using
va-alert
- added comments to distinguish the different components

* clean up:
- remove redundant reducer
- remove redundant code

* update import paths for dashboard e2e tests

* fixed component import path

* - made logic for displaying notifications to be
more scalable

* remove unecessary eslint-disable

ndsprinkle committed:
Add relationship to veteran web component pattern to the forms library (#24854)

* Create new relationship to veteran pattern and add to mock form

* Use updateSchema to conditionally and relatively make the relationshipToVeteranOther field required

* Add expandedContentFocus param to ExpandingGroup to support indenting and focusing on expanded under field

* Use radio pattern UI and schema

* relationshipToVeteranOther property name change to fix validation issue

* Refactor expandedContentFocus property to be used with the expanding field

* Update tests for relationshipToVeteran

* Update ExpandingGroup to properly focus inside the expanded field regardless of widget, react, or web component

* Update pattern to support passing in a specific person title instead of the entire titles and update labels to be possessive and use the specific person title

* Update types to include expandedContentFocus

* Fix mock form test

* Rework veteran title logic into a const

randimays committed:
VACMS-13750 Income Limits: add real content (#24809)

rhasselle-oddball committed:
error message updates (#24940)

josephrlee committed:
[FSR] Review page spacing and font-sizes fixes (#24941)

mattwrightva committed:
MHV-42360: Pdf scaffold helper created (#24914)

* MHV-42360: Pdf scaffold helper created

* MHV-42360: Moved helpers to MHV shared and adjusted params

* MHV-42360: Pdf helper function testing complete

* MHV-42360: tweaks and fixes

* MHV-42360: unit test fix

---------

Co-authored-by: wright0776 <matwright2010@gmail.com>

gmrabian committed:
Simple forms: Content updates to 21P-0847 (#24841)

* content updates to 21P-0847

* fix spacing on confirmation

* update title on all pages

* confirmation page next step text change

* update tests for new form flow

* rename rjsf to 'view:etc' pattern for non-field content

* add comma to text

* update to use new relationship to veteran pattern

---------

Co-authored-by: Garrett Rabian <garrett@coforma.io>

ataker committed:
Issue with update, revert to previous version for now (#24949)

ataker committed:
Replace select react component with web component (#24951)

* replace react component with web component and update tests

* Fix options

tlei123 committed:
Fix 21-10210 statement JSON-parse bug (#24956)

allisonlu committed:
added date-time prop to va-notification (#24955)

ataker committed:
Replace text input react component with web component (#24954)

* Replace text input react component with web component

Add functionality to check if field is "dirty" before running validation

* Standardize language around whether field dirty

Commit:
fix: delete flakey tests (#24958)

jamigibbs committed:
Add analytics mapping for va-modal button click (#24944)

* Add analytics mapping for va-modal button click

* Revert event name change

* Update ga4 mapping names

rmessina1010 committed:
Fixing semver vulnerabilities (#24957)

* semver upgrade, first pass

* updates babel-plugin-polyfill-corejs3 (no effect)

* updates helper-define-polyfill-provider

* updates @babel/preset-env in lock file;  updating package.json still causes fail

rocio-desantiago committed:
Supplemental Claims | Refactor code with va-link (#24923)

* refactor DownloadLink with va-link component

* update unit/cypress tests for va-link

---------

Co-authored-by: Rocio De Santiago <rociodesantiago@CoformaSantiago.lan>

elwoodva committed:
[MHV-45214] Changing Message Action Buttons/Reply (#24789)

* [MHV-45214] Changing Message Action Buttons/Reply

* [MHV-45214] Button clean-up

* [MHV-45124] Change section to div

* [MHV-45214] Changing section to div

* [MHV-45214] Show top reply on Sent

* [MHV-45124] Change cypress test

* [MHV-45214] Added 508 header that was missed

* [MHV-45214] Removed label

* [MHV-45214] Using reply-button-body not top

* [MHV-45214] added margin between buttons

* [MHV-45214] Changed small to xsmall

* [MHV-45214] Removed class vads-u-margin-y--4

Thrillberg committed:
Update form 21-0972 route (#24959)

Mottie committed:
Decision Review | Clean up review & submit page (#24964)

* Change issue review page titles

* Fix review page line height and edit alignment

* Remove issue extra header & description in review mode

* Update NOD area of disagreement headers

* Hide empty "Something else" entries

* Update unit test

KyleCardwell committed:
Mhv 45263 compose form updates (#24918)

* updated edit ilst button to secondary va-button

* updated buttons, styling, layout, logic for displaying Delete button

* adjusted failing cypress and unit tests

* fixed failing cypress test pushSendMessageWithKeyboardPress

* fixes from PR comments

* fixed testing issues with styling

adam-antonioli committed:
MBMS-40582 Self Service: Inaccurate validation on Review Application page (#24950)

* MBMS-40582 change pre-need form military history to not use depends conditional

* MBMS-40582 add other attempts to change pre-need form military history to not use depends conditional

* MBMS-40582 add simple redux implementation

* MBMS-40582 revert all previous commits for branch MBMS-40582-self-service-inaccurate-validation-on-review-application-page

* MBMS-40582 change form review page section on change error message and add prod flag

mattwrightva committed:
MHV-45442: Care summaries and notes mhv data hookup complete (#24948)

* MHV-45442: Care summaries and notes mhv data hookup complete

* MHV-45442: Mock data fixed

* MHV-45442: unit test fixes

* MHV-45442: Unit tests fixed

* MHV-45442: checks added to notes and vaccines converter functions

---------

Co-authored-by: wright0776 <matwright2010@gmail.com>

vbahinwillit committed:
fix 'invalid prop' warning message (#24939)

mmoyer-va committed:
[#MHV-45445] Updated text to match latest designs (#24975)

randimays committed:
VACMS-13750 Make all Income Limits links relative (#24972)

mattwrightva committed:
MHV-42359: chem hem pdf generation complete (#24981)

Co-authored-by: wright0776 <matwright2010@gmail.com>

longmd committed:
Update HCA expenses question content (#24974)

longmd committed:
Update Forms Library legends to meet Design System standards (#24969)

jenniemc committed:
Fix prop types warning messages (#24962)

chri5tia committed:
VACMS-14119: GI Bill Redirect (#24982)

KyleCardwell committed:
changed padding and margins around dropdown menu and breadcrumb (#24976)

vbahinwillit committed:
60043 Update the breadcrumb on the Appointments landing page (#24936)

* update breadcrumb on appointment page

* modified per code review

Mottie committed:
NOD | Add part III questions (#24926)

* Add NOD part 3 feature toggle

* Add part III questions

* Add tests

* Add countryCodeISO2 value to submissions

* Add submit to v2 endpoint w/feature toggle

* Update tests

* Remove submitUrlV2

allisonlu committed:
remove check for payment history (#24980)

(this should not affect whether or not these sections should show)

KolbyVA committed:
[#mhv-46753] styling/content updates (#24983)

Dr-Pongo committed:
[FSR] 61841 - Adding fetch to get gmt threshold (#24933)

KyleCardwell committed:
Mhv 47123 remove draft interstitial (#24952)

* changed main to div, removed interstitial component from ThreadDetails

* unit and cypress tests

ndsprinkle committed:
Add support for the v3 segmented progress bar to the forms library (#24978)

* Add support for the v3 segmented progress bar, currently only enabled with a formConfig property

* Updates types with new formConfig property

* Updates forms unit test with new property

longmd committed:
Update HCA footer text (#24989)

longmd committed:
Increase screenreader support for HCA fieldset titles & descriptions (#24988)

mmoyer-va committed:
MHV-44887 Hooked up backend for VHIE sharing (#24990)

* [#MHV-44887] Hooked up backend for VHIE sharing

* [#MHV-44887] Improved error handling

longmd committed:
Update HCA field descriptions to match design system standards (#24973)

longmd committed:
Update HCA dependents list card to meet design system standards (#24971)

fazilqa committed:
Mhv 47456 sent messages page (#24984)

* create PatientMessageSentPage.js

* generalized methods in PatientMessageTrashPage.js

* add sorting test in sent folder

* upd sorting test in sent folder

dellerbie committed:
Fixes My VA Lighthouse claims status text (#24960)

tlei123 committed:
21 10210 - Update Confirmation-page content (#24947)

* Disable 21-10210 local-dev initial-data

* Update 21-10210 Confirmation-page content

s-caso committed:
Sa 58801 email phone duplicate (#24192)

* add custom email component

* add custom phone component

* add duplicate contact info modal

* upadte actions and reducer

* use custom email/phone components in form

* update global application state and connect to formData

* update hard coded value

* add alerts and remove radio options if dupes

* update modal text conitions and alert conditions

* update text on modal

* update mapStateToProps call

* update hideIf logic

* add missing link

* update link

* update filter statements

* refactor if/else statement

* fix typo

* check for string value 'true' add link

* fix typo

* update wording and hide radio if both duplicates

* escape validation check if both email and phone are dupe

* conditionally render radio buttons

* pass values to duplicate contact call

* add feature flag and wrap handlechange calls in them

* wrap initial call in feature flag

* correctly call feature toggle

* update isDupe key to dupe

* switch true/false from string to boolean

* fix typo

* use boolean values not strings for comparisons

* add dupe info to formData

* initial values load before call to backend

* use correct order in duplicate call

* populate dup call info in formData

* text update for alert

* update formData mobilephone

* update formData values that review form uses

* use updated mobile phone path

* update link text

* wrap duplicate call in feature flag

* use formdata feature flag

ediiotero committed:
Ask VA support for Auth (#24963)

* using auth in ask va mock up

* fix lint errors

---------

Signed-off-by: Eddie Otero <eddie.otero@oddball.io>

mmoyer-va committed:
MHV-44887 Fixed Settings Title (#24993)

* [#MHV-44887] Fixed h1

* [#MHV-44887] Fixed breadcrumbs as well

OptionSelect committed:
60628 - EFSR - Organize Components Folder (#24961)

* Folderize all free floating components, form still submits successfully, removed expandinggroup bc of old experience and blocking linting

* Fix fsr cypress

* Fix broken unit tests

* Move remaining missed files per Andrew

rhasselle-oddball committed:
update form margins for ObjectField and FormNavButton (#24938)

* update form margins for simple forms using web components
* add classNames for ObjectField div and SaveLink container

longmd committed:
Update Household V2 URL routes to more closely match content (#24986)

dependabot[bot] committed:
Bump moment-timezone from 0.5.34 to 0.5.35 (#22076)

* Bump moment-timezone from 0.5.34 to 0.5.35

Bumps [moment-timezone](https://github.com/moment/moment-timezone) from 0.5.34 to 0.5.35.
- [Release notes](https://github.com/moment/moment-timezone/releases)
- [Changelog](https://github.com/moment/moment-timezone/blob/develop/changelog.md)
- [Commits](https://github.com/moment/moment-timezone/compare/0.5.34...0.5.35)

---
updated-dependencies:
- dependency-name: moment-timezone
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump moment-timezone from 0.5.34 to 0.5.35

Bumps [moment-timezone](https://github.com/moment/moment-timezone) from 0.5.34 to 0.5.35.
- [Release notes](https://github.com/moment/moment-timezone/releases)
- [Changelog](https://github.com/moment/moment-timezone/blob/develop/changelog.md)
- [Commits](https://github.com/moment/moment-timezone/compare/0.5.34...0.5.35)

---
updated-dependencies:
- dependency-name: moment-timezone
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

---------

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Ian <harrison.ian.t@gmail.com>

longmd committed:
Fix HCA Deductible Expenses legend formatting (#24999)

longmd committed:
Update HCA Need Help content (#24997)

rmessina1010 committed:
bumps word-wrap to 1.2.5 (#25000)

fazilqa committed:
Mhv 43996 custom folder page (#25003)

* custom folder page

* add methods for custom folder

* add mock responses for custom folder

* add tests for custom folder

Thrillberg committed:
Fix error messages in simple form 21P-0847 (#24968)

* Fix error messages in simple form 21P-0847

* Add one last error message

* Handle further change requests

s-caso committed:
check for blank string as well as if false value (#25002)

* check for blank string as well as if false value

* don't show alerts if dup email/phone not present

jsimonVA committed:
#VFEP-686 #comment removed flags and deleted unused Unit Tests (#24917)

data-doge committed:
[61817] Set up NOD RUM dashboard (#24970)

* Sketches

* Add feature flag

* Init RUM

* add back environment function

* Remove comments

---------

Co-authored-by: Robin Garrison <robin.garrison@adhocteam.us>
Co-authored-by: Rob Garrison <wowmotty@gmail.com>

ericvoshall committed:
Support message count on MHV landing page (#24791)

* Support message count

* Only display message count if at least 1

* Remove console.error

* Linting fixes

* Import path update

* Add indicator

* Tests

* Fix mobile style issue

asg5704 committed:
moves placement of authReturnUrl call (#24987)

rmessina1010 committed:
Update node-jsonpointer to 5.0.0 (#25007)

jenniemc committed:
Add optional chain (#24995)

Mottie committed:
Remove decision date limitation with feature flag (#25013)

ryanshaw committed:
53091 remove getCommunityCareData v0 function (#24901)

* Fixes test

Test was failing when run in the Pacific Time Zone

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Updates mocked date on test

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Typo

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Revert "Updates mocked date on test"

This reverts commit 21cde9bc64fbe116638ef8aeb732e7592e6ce875.

* Replaces getTeimzonTestDate function for unit test mocks

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Removes getCommunityCareData function

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Revert "Replaces getTeimzonTestDate function for unit test mocks"

This reverts commit d2c42c631e1d62f61976c3c4aafbbf0ae80b9de0.

* Revert "Revert "Updates mocked date on test""

This reverts commit 0cf8351a0857429b848e850583fe82ee1066165d.

* Revert "Updates mocked date on test"

This reverts commit 21cde9bc64fbe116638ef8aeb732e7592e6ce875.

* Revert "Fixes test"

This reverts commit ab764d9dab9b0883b5a56562619be17b1a0e4e00.

---------

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

ryanshaw committed:
Removes VAOS v0 transformer unit tests (#25011)

* Removes old v0 transformer unit test

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Renames appointment service unit test

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

---------

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

ryanshaw committed:
Replaces getTimezoneDate mock helper function in VAOS unit tests (#24932)

* Replaces getTimezoneDate mock helper function

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Adds comments

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Removes more instances of getTimezoneTestDate

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Updated comments

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Removes getTimezoneTestDate

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

---------

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

Commit:
Remove VAFSC mvp code (#25006)

* remove VAFSC mvp code

* chore: clean up imports

* fix: @dept imports weren't working for e2e tests

* fix: ~ not working in platform

* fix: one more time

Mottie committed:
NOD | Make code more readable (#25014)

* Make code more readable

* Fix unit test

data-doge committed:
Enable `trackUserInteractions` for NOD RUM. (#25015)

This is a requirement for DataDog RUM's "heatmap" feature. We think that design/research will find this feature helpful. This feature also requires us to upgrade @datadog/browser-rum to >4.40. @micahaspyr will be performing that upgrade in a separate PR, coming soon.

adam-antonioli committed:
MBMS-42390 change pre need get help section content (#24996)

* MBMS-42390 change pre need get help section content

* MBMS-42390 add prod flag

fazilqa committed:
add request body validation (#25019)

Asterix22 committed:
Mhv 46192 drop down list page (#25017)

* Validate DropDown on ListPage

added scripts to verify the drop down functionality on Medications List Page

* Added updates for page objects

updated scripts to include page objects and changes to one of the element locators

* Update LandingPage.jsx

* Update Medications List Page

changed the element locator to data-testid

---------

Co-authored-by: raji.venkatesh <raji.venkatesh@accomsolutions.com>

s-caso committed:
Dup email phone hotfix (#25021)

* only check for duplicate contact info if valid phone/email

* check formData in useEffect

yonidoronpeters committed:
[DHP-916] WIP - Add disclaimer text (#25009)

* [DHP-916] WIP - Add disclaimer text

Co-authored-by: Yoni Doron-Peters <jonathan.d@thoughtworks.com>

* DHP-916 Add <p> tags to match separation between paragraphs like in the wireframes

* DHP-916 Extract disclaimer text to new method per PR comment

* DHP-916 remove extra <div/> tag

---------

Co-authored-by: Mariah Sager <mariah.sager@thoughtworks.com>

bradbergeron-us committed:
Bugfix: Sa 62576 fix action link spacing (#25022)

* removes action link customization in enrollment verification

* testing out custom paddings on action links

* Removes custom action link styling

rmessina1010 committed:
Update jsonwebtoken to fix vulnerability (#25027)

ryguyk committed:
Updates proxy-rewrite documentation. (#25032)

fazilqa committed:
Mhv 47275 add sorting tests (#25033)

* add sorting validation for trash-folder spec

* update sorting validation for sent-folder spec

* update sorting validation for trash-folder spec

* update sorting validation for inbox, drafts and custom folder spec

* update test structure

KyleCardwell committed:
decreased top margin on draft, compose and reply message headings (#24998)

* fixed top margin on draft and compose message headings

Commit:
MBMS-42191 Removed Prod Flag (#25030)

ndsprinkle committed:
Re-enable cypress tests and change review page tests to support additional params (#25020)

rmessina1010 committed:
updates mapbox-sdk and schema-util (#25038)

ryanshaw committed:
Removes v0 appointment transformers (#25018)

* Removes v0 appointment transformers

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Renames v2 appointments transformer

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

---------

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

bradbergeron-us committed:
Sa 72693 remove labels (#25039)

* Removes sponsor I've added label on review pages

* Removes Someone Not Listed as an Option on Sponsor Selection

* Adds Proptypes to SponsorsCheckboxGroup component

* Adds Props Types to SponsorsSelectionPage.jsx to address linter complaints

s-caso committed:
sync email form data with input (#25040)

RajiVenkatesh18 committed:
Validate DropDown on Details Page (#25042)

Added scripts to verify drop down functionality on medications details page

Co-authored-by: raji.venkatesh <raji.venkatesh@accomsolutions.com>
Co-authored-by: Aaron Read <Aaron.Read@va.gov>

jenniemc committed:
Update breadcrumb on confirmation page (#25028)

* Change back link text

* fix mock appointment time selection

Mottie committed:
NOD | Update to v1 submission endpoint (#25043)

* Update NOD v1 submission endpoint

* Update e2e tests

Midge-dev committed:
added aria-describedby to description (#25008)

Mottie committed:
NOD | Add masks and hide PII for Datadog RUM recordings (#25023)

* Hide contact info fields in datadog recordings

* Hide file name in datadog recordings

* Add Datadog mask/hidden classes for PII in recordings

* Update unit tests

* Add common mask/hidden fields in 995

* Add common mask/hidden fields in 996

* Add missing platform tests

* Add mask to Veteran name

* Hide Veteran name & gender

* Hide uploaded file name

s-caso committed:
update mobile phone in customer component (#25045)

ryanshaw committed:
Removes v0 slot transformers  (#25026)

* Removes v0 slots transformers unit tests

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Removes v0 slot transformers

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Renames v2 transformer file

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

---------

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

ryanshaw committed:
Removes v0 healthcare service transformers (#25024)

* Removes v0 healthcare service transformers

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Removes unit tests for v0 healhcare service transformers

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Renames v2 transformers file

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

---------

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

ryanshaw committed:
Removes v0 location transformers  (#25025)

* Removes v0 location transformer functions

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Removes v0 location transformer unit tests

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Added functions still used to the v2 location transformers

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Commenting this out, the entire file is removed in another open PR, allows tests to pass

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Remames v2 transformers file

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Temporarily fixes import

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

* Revert "Temporarily fixes import"

This reverts commit 1d9a8dab08b6d42d36d2a72b202455617f9e210f.

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

---------

Signed-off-by: Ryan Shaw <ryan.shaw@adhocteam.us>

KolbyVA committed:
[#mhv-45377] refill button functionality and error messages (#25048)

Mottie committed:
NOD | Fix nested fieldsets on evidence page (#25046)

* Add showFieldLabel "no-wrap" setting

* Add hideDuplicateDescription ui option

* Fix nested fieldsets & duplicate descriptions

rd-huerta committed:
[department-of-veterans-affairs/va-virtual-agent/#990] Enable speech for the RX skill


Co-authored-by: helene-mccarron <helene.mccarron@thoughtworks.com>
Co-authored-by: Evan Ubiera <evan.ubiera@thoughtworks.com>

ndsprinkle committed:
Revert expanded content focus on field input (#25016)

* Revert special focusing of expanded field

* Re-adding focus on the outer element because the text input needs to be 'one tab away'

* Revert "Re-adding focus on the outer element because the text input needs to be 'one tab away'"

This reverts commit f0a5a0aeea964c03e824184d5a4838078e3c9e83.

* Disable mock data in cypress

* Switch from realType to type

Wyzardsleeves committed:
MHV-42361 Added download feature to Pathology.jsx (#24913)

* added download feature to Pathology.jsx

* removed hardcoded code and added results

Wyzardsleeves committed:
MHV-42544 added PDF Download for Notes (#25004)

* added PDF Download for Notes

* removed added by justin

* updated the pdf generation function with generatePDFScaffold

tlei123 committed:
Simple-forms - Front-end form-submission JSON.parse bugfix (#24965)

* Move JSON-stringify/-parse bugfix from form-app to form-system helper

* Refine forms-system stringifyFormReplacer backslashes regex

* Move JSON.parse bugfix from forms-system helper to shared submit-transformer

* Fix JSON.parse bugfix regex

* Refactor JSON.parse bugfix regex

* Update shared submit controller to filter out escaped characters

* Removed replacement for forward slash and convert back slashes to forward slashes

---------

Co-authored-by: Nick Sprinkle <ndsprinkle@gmail.com>

jerekshoe committed:
[CST] Adding unit tests for DDL link (#24943)

* Adding unit tests for DDL link

* Marking showClaimLettersLink prop as required for relevant components

mattwrightva committed:
MHV-42360: Microbio pdf fix (#25060)

Co-authored-by: wright0776 <matwright2010@gmail.com>

fazilqa committed:
add verify backToMessages btn method (#25061)

chriskim2311 committed:
add redirect for education rates page (#25065)

Co-authored-by: Chris Kim <chriskim@Chriss-MacBook-Pro.local>

Mottie committed:
Supplemental Claim | Update date validation (#25062)

* SC: Fix invalid decision date entry

* SC: Add tests

KolbyVA committed:
[#mhv-43488] updated breadcrumbs and removed "/health-history" (#25064)

Mottie committed:
Add aria-describedby to evidence radios (#25063)

Mottie committed:
NOD update date validation (#25067)

rocio-desantiago committed:
Update NeedsToVerify for all 3 forms and create new shared folder (#25034)

Co-authored-by: Rocio De Santiago <rociodesantiago@CoformaSantiago.lan>

rhasselle-oddball committed:
Don't error on blur if fields are empty (#25031)

* don't error on blur if fields are empty

Commit:
Focus for address edit view and validation alert (#24985)

* fix: focus for address edit view and validation alert

* test: fix focus e2e, remove uneeded alert focus

* chore: cleanup comments

* fix: typo

* fix: only focus on edit button when no forceEditView

* fix: undo previous patch work

* fix: focus when cancelling address validation

* fix: tiny typo for sign-in

Mottie committed:
NOD: submit email for v1 endpoint (#25069)

* NOD: submit email for v1 endpoint

* Remove comment

* Fix JSDoc

fazilqa committed:
fix failing tests (#25074)

robertroddev committed:
[MHV-45014] no fields needed update with new api endpoint, however some of the prescription details page did. (#25059)

Mottie committed:
NOD | Add more Datadog hidden content (#25073)

* Add more Datadog hidden content

* Mask address confirmation page

* Fix breaking unit tests

helene-mccarron committed:
[#department-of-veterans-affairs/va-virtual-agent#1096] Use entry message as RX entry (#25080)

jamigibbs committed:
Update component library to v18.0.0 (#25081)

dellerbie committed:
fix: Update My VA Benefit Payments card headings (#25083)

mattwrightva committed:
MHV-42360: micriobio pdf date fix in header (#25076)

it-harrison committed:
remove references to react CheckboxGroup component in eslint and analytics (#24886)

Mottie committed:
NOD & HLR: Fix disagreement title & a11y (#25086)

dellerbie committed:
fix: update Benefits payments headings to use design system css (#25088)

helene-mccarron committed:
[#department-of-veterans-affairs/va-virtual-agent#990] Change send box text (#25082)

Co-authored-by: Evan Ubiera <evan.ubiera@yum.com>

joehall-tw committed:
updates the uri to use the new, modular api endpoint (#25049)

chriskim2311 committed:
Add new semantic h2 with h4 visuals for discharge wizard (#25052)

Co-authored-by: Chris Kim

Asterix22 committed:
Mhv 46191 print medications on list page (#25084)

* Validate Print List on List Page

Added scripts to verify Print List on List Page

* Verify Print Medications on List Page

Added changes to the method name

* Update Navigate to Message Details

Test Fixes to address failures and rerun checks

* Update secure-messaging-folders.cypress.spec.js

---------

Co-authored-by: raji.venkatesh <raji.venkatesh@accomsolutions.com>
Co-authored-by: raji.venkatesh <114511630+RajiVenkatesh18@users.noreply.github.com>

Asterix22 committed:
Mhv 46199 cypress test maintenance (#25094)

* Update secure-messaging-keyboard-nav-to-message-details.cypress.spec.js

* Update secure-messaging-keyboard-nav-to-message-details.cypress.spec.js

* Update secure-messaging-keyboard-nav-to-message-details.cypress.spec.js

* Update secure-messaging-keyboard-nav-to-message-details.cypress.spec.js

* Update secure-messaging-keyboard-nav-to-message-details.cypress.spec.js

---------

Co-authored-by: haniasma <121971510+haniasma@users.noreply.github.com>
Co-authored-by: Ray Messina <48040208+rmessina1010@users.noreply.github.com>

longmd committed:
Remove duplicate identity fields for guest users from Health Care Application (#25056)

s-caso committed:
check endDates on tours of duty and update data if none is present (#24616)

* check endDates on tours of duty and update data if none is present

* check if active duty

longmd committed:
Update error message keys for Forms Library emailUI definition (#25091)

helene-mccarron committed:
[#department-of-veterans-affairs/va-virtual-agent#990] Fix send text and change selector

longmd committed:
Update HCA transform function to add in veteran information (#25100)

Mottie committed:
NOD: Fix older decision date message (#25092)

Mottie committed:
Hide more PII in Datadog RUM (#25096)

Commit:
Profile/aw/hotfix eslint upgrade errors (#24369)

* fix: wrap sytlelint in function

* fix: remove invalid whitespace chars

* fix: eslint errors

* fix: remaining eslint errors

* fix: spacing for bolded sections

* fix: last spacing issue

dellerbie committed:
fix: bolden My VA benefits payments header (#25101)

acald-creator committed:
replace babel-eslint with @babel/eslint-parser

longmd committed:
Pull Veteran name from view field on HCA confirmation page (#25102)

fazilqa committed:
fixing failed specs (#25106)

adam-antonioli committed:
MBMS-43474: Self-Service: Pre-Need: Introduction - Veteran and Non-Veteran Flow: Content Changes (#25070)

* MBMS-43474 change content on pre feed form introduction

* MBMS-43474 add prod flag

* MBMS-43474 remove a gap between paragraphs in preneed intro page

elwoodva committed:
[#MHV-45213] Sent Page Updates (#24967)

* [#MHV-45213] Sent Page Updates

* [MHV-45213] Changes from code review

* [MHV-45213] Updated cypress test

* [MHV-45213] Using useState

* [#MHV-45213] Just using folder name

* [MHV-45213] Fixed linting issues

* [MHV-45213] Removed linting fixes

* [#MHV-45213] Sent Page Updates

* [MHV-45213] Changes from code review

* [MHV-45213] Updated cypress test

* [MHV-45213] Using useState

* [#MHV-45213] Just using folder name

* [MHV-45213] Fixed linting issues

* [MHV-45213] Removed linting fixes

* [MHV-45213] Removed " messages"

* Update FolderHeader.jsx

* Update FolderHeader.jsx

* [MHV-45213] Removed 'messages'

* [MHV-45213] Changed 'Sent messages' to 'Sent'

* [MHV-45213] Passing Cypress test

* [MHV-45213] Removed 'messages'

* [MHV-45213] Only put Compose link on Inbox

* [#MHV-45213] Sent Page Updates

* [MHV-45213] Changes from code review

* [MHV-45213] Updated cypress test

* [MHV-45213] Using useState

* [#MHV-45213] Just using folder name

* [MHV-45213] Fixed linting issues

* [MHV-45213] Removed linting fixes

* [#MHV-45213] Sent Page Updates

* [MHV-45213] Changes from code review

* [MHV-45213] Using useState

* [MHV-45213] Fixed linting issues

* [MHV-45213] Removed linting fixes

* [MHV-45213] Removed " messages"

* Update FolderHeader.jsx

* Update FolderHeader.jsx

* [MHV-45213] Removed 'messages'

* [MHV-45213] Changed 'Sent messages' to 'Sent'

* [MHV-45213] Passing Cypress test

* [MHV-45213] Removed 'messages'

* [MHV-45213] Only put Compose link on Inbox

* [MHV-45213] Update secure-messaging-folders.cypress.spec.js

* [MHV-45213] Removed spaces

---------

Co-authored-by: Alex Morgun <87077843+oleksii-morgun-va@users.noreply.github.com>

kjsuarez committed:
add feature flag for debts-api module (#25108)

rhasselle-oddball committed:
Add mock form which mimics ux form patterns only using v3 web components (#25089)

* add mock form which mimics the ux forms patterns

Commit:
MBMS-44008 Self-Service Pre-Need - Header Issues on Review Page Cards (#25079)

* MBMS-44008 Added Titles to review cards

* MBMS-44008 Changed capitalization of service periods

rhasselle-oddball committed:
update phone pattern (#25090)

* update phone pattern

helene-mccarron committed:
[#department-of-veterans-affairs/va-virtual-agent#990] Fix send text … (#25104)

rmessina1010 committed:
udates request to address vunlerability (#25055)

OptionSelect committed:
61414 - [FSR] Remove Combined FSR feature flag (#25012)

* Remove combined from assets

* Fix rec vehicles urls

* Remove combined resolution options

* remove combined vet info section

* Remove fsr flag from transform

* Bring download button back!

* Fix one test and undefined on download

* Fix unit tests with Andrew

* fix cypress tests with Andrew

* Unskip alert cards

* Fix fetchsuccess

* Rename tests

* Address selectedDebts existence

* Consolidate maximals and minimals

* Remove no longer used files and selectedDebts reference

* Fix additional combined remnants

* Remove CFSR from transform unit

* Remove remaining cfsr flags and remove cfsr transform logic tests

jamigibbs committed:
Update component library version (#25117)

tlei123 committed:
Ignore mochawesome-report compiled-stylesheets in stylelint-config (#25099)

jenniemc committed:
Add optional chaining to startsWith method (#25072)

* Add optional chaining to startsWith method

* Changes from review

jenniemc committed:
Feature/61015 revert back breadcrumb text for pending appointments (#25112)

* Revert to previous breadcrumb text

* Fix for mock request appts when slots.id is null

adam-antonioli committed:
MBMS-41711 remove prod flag for MBMS 40582 (#25114)

adam-antonioli committed:
MBMS-45605 remove prod flag for MBMS 42390 (#25115)

robertroddev committed:
[MHV-47640] prescription details updates (#25116)

* [MHV-47640] added last filled on date, realigned all the pages to VA logo, and added made it so the  details page always brings the user to the top when it loads.

* [MHV-47640] changed the method used to scroll the page up on mount, now using a focuselement for better screen reader experience.

Commit:
Edit as a subtask midpoint updates (#25051)

* feat: add cancel confirm modal, context for edit, breadcrumb

* feat: extract breadcrumb, update content

* test: add unit for breadcrumb, cleanup

* rename unit test

* fix: clear state of edit modal on cancel

* chore: comment cleanup

* chore: cleanup mock server

* test: add e2e for edit page

* feat: add mock POST and PUT for email address updates

* fix: revert export structure of user mocks slightly

* test: fix e2e tests

* fix: update more user objects in tests

* fix: correct prop types

alex-morgun-1 committed:
Mhv 47574 message signature (#25087)

* [MHV-47574] api call to get signature data

* [MHV-47574] messaging pref reducer + getSignature action

* [MHV-47574] patient safety banner consolidation

* [MHV-47574] code cleanup

* [MHV-47574] adding signature on compose and reply views

* [MHV-47574] Compose unit tests

* [MHV-47574] update signature route

CBonade committed:
E2e allow blocking check (#25066)

* test file path collection

* test blocking matching logic to disallowed spec path list

* test blocking matching logic to disallowed spec path list

* test blocking matching logic to disallowed spec path list

* test blocking matching logic to disallowed spec path list

* Test

* Test

* Test

* Move logic around

* Move logic around

* Fix incorrect step job reference

* Disable linting rule

* try and run with known failure

* WIP

* Test with code change that is not currently under warning status

* Try template page that has blocked tests

* Tidy code"

* remove console logs

---------

Co-authored-by: Curt Bonade <cbonade@Curts-MacBook-Pro.local>

longmd committed:
Allow for resubmission of ID Form in Health Care Application (#25123)

KyleCardwell committed:
Mhv 47124 unread sent messages (#25036)

* updated logic to now show message as unread if in sent folder or it has been read

* added unit tests for sent and read messages

* added new constant for message status to be used in logic for isSentOrRead

KyleCardwell committed:
Mhv 45255 trash updates (#25097)

* added new state variable and control for End of Conversations text visibility

* e2e and unit tests for end of conversations text

* added styling to move sort button to new line on small screens

* adjusted sort button and select to fill width on small screens

---------

Co-authored-by: Adrian Rollett <adrian.rollett@va.gov>

delli-gatti-michael committed:
Mbms 45590 preneed e2e updates (#25078)

* MBMS-45590 fix test data and reenable sponsor test

* added required field test file and edited test data

* added helper file and required field checks for sponsor info page

* add checks for all required fields, add mock feature flag info

* moved axe checks

* create new test data file

* fix unit test

* add helper file and begin refactoring all preneed tests

* add function to fill applicant information page

* add military history, benefit selection, and previous name helper functions

* add contact info and submit helper functions

* move errorCheck function, formatting cleanup

josephrlee committed:
[eFSR] Made debts plural on personal statement page (#25118)

Commit:
fix: eslint (#25129)

CBonade committed:
Remove conditional dependency install (#25131)

Co-authored-by: Curt Bonade <cbonade@Curts-MacBook-Pro.local>

simiadebowale committed:
#59733: Update the URL and title page on the Appointments landing page (MHV on VA.gov) (#24994)

* updated url

* updated root url

* updated entry name

* updated e2e test

* updated entry name

brianseek committed:
updated word "certify" to "confirm" (#25133)

tlei123 committed:
Finalize 21-10210 content (#25135)

amponce committed:
Data transform income calculator (#25053)

* breakout-income-calculator

* unit tests and simplify function call

* cleanup comments

* cleanup naming

* cleanup and comments for readablity

* naming and cypress

* add default values to prevent errors when there is no gross salary

* change to getMonthlyIncome in comments

* skip failing cypress test

Commit:
Add additional description to gender identity of Personal Information page (#25124)

* feat: add description to gender identity

* test: remove feature toggle info card test

* fix: prop types

* chore: rename as cardFields, update contacts import

mattwrightva committed:
Mhv 45440 allergies data hookup (#25130)

* MHV-45440: Mapped test data swapped for likely response data

* MHV-45440: Allergies MHV data hookup work done

* Update unread.message.count.cypress.spec.js

---------

Co-authored-by: wright0776 <matwright2010@gmail.com>
Co-authored-by: Aaron Read <Aaron.Read@va.gov>

alex-morgun-1 committed:
MHV Update FolderHeader unit test (#25139)

* Update FolderHeader.jsx

* Update unread.message.count.cypress.spec.js

---------

Co-authored-by: mattwrightva <107576133+mattwrightva@users.noreply.github.com>

mattwrightva committed:
MHV-45441: Chem/hem mhv data hookup work done (#25138)

* MHV-45441: Chem/hem mhv data hookup work done

* MHV-45441: Added todo comment for error handling

---------

Co-authored-by: wright0776 <matwright2010@gmail.com>

jenniemc committed:
Update breadcrumb text for past appointments page (#25134)

Co-authored-by: Simi Adebowale <47654119+simiadebowale@users.noreply.github.com>

Commit:
Remove use_profile_info_card toggle work (#25137)

* chore: remove old table component and toggling

* chore: remove toggle from platform and mock server

* test: remove feature toggle info card test

* test: clean up unit test for removed toggle

* fix: remove toggle from fixture

vbahinwillit committed:
61301 Cerner Re-direct on the Appointments homepage (#25068)

* fix to improve Cerner redirect

* fix undefined exceptions

* fix to check for cerner overrides

* remove deprecated cerner function

* fix to add GA event #62997

* fix language for single Cerner facility

christinec-fftc committed:
59003 Prod launch of first pass of 526 Paper Sync (#25132)

va-albers committed:
Update Intent to File EVSS error message (#25147)

* Update Intent to File EVSS error message

Update EVSS error message for 526 Intent to file workflow.  Because of impending PACT ACT issues and EVSS errors a change is being made to ensure that veterans understand that their request has been received even if they have an EVSS error issue.

* Update itfWrapper.jsx

close br element

* remove unused import

* update unit test text

---------

Co-authored-by: Mike Chelen (he/him) <michael.chelen@va.gov>

Dr-Pongo committed:
[FSR] 61839 - Fix for total income calulation #25146

simiadebowale committed:
added feature flag (#25136)

Dr-Pongo committed:
[FSR] Streamlined waiver chapter changes (#24937)

rhasselle-oddball committed:
add mock form for header testing (21-0845 based) (#25140)

ndsprinkle committed:
Add new Date of Death web component pattern variation and implement in 21P-0847 (#25125)

* Add date of death web component pattern and implement in 21P-0847

* Revert old email definition change

* Fix email unit test

tblackwe committed:
Updating FE to enforce a 99MB limit on PDF uploads (#25152)

kjduensing committed:
Change max file size to 99MB. (#25153)

Helps EVSS cope with the increased user load.

s-caso committed:
show all options for radio buttons if a value exists and show alerts … (#25141)

* show all options for radio buttons if a value exists and show alerts for dupe issues no errors

* check for mobile phone and update wording of alert

* update formData mobile phone in custom phone component

Dr-Pongo committed:
[FSR] Fix - Adding spread for additional income #25155

randimays committed:
VACMS-14593 Fix Income Limits results page accordion header levels (#25110)

allisonlu committed:
removed logic so Learn how to file a claim link (#25143)

always shows

Commit:
fix: only add margin in section headings (#25154)

Dr-Pongo committed:
[FSR] 61844 - SW - Adding Cash on Hand validation and review page #25150

s-caso committed:
mobilePhone no longer having value reset (#25161)

* mobilePhone no longer having value reset

* remove commented out code

* set current phone value in dupe check for custom emai component

Dr-Pongo committed:
[FSR] SW Updating streamlined calculations (#25159)

alex-morgun-1 committed:
Mhv send button analytics (#25164)

* reinstate va-button on message send form for analytics

* refactoring compose action buttons + cleanup

* addressing linter issues

* styling tweaks

amponce committed:
add streamlined flag to transform (#25166)

* add streamlined flag to transform

* remove comments and console log

OptionSelect committed:
63263 - [SW] - E2E Testing (#25173)

* Add long and short e2e testing

* Re-add short minimal

* Fixing additional income input

* Add long and short e2e testing

* Re-add short minimal

* Fixing additional income input

* Fixing long bath with accurate data

---------

Co-authored-by: Andrew Rodiek <andrew.rodiek@gmail.com>

mattwrightva committed:
MHV-45443: Vitals mhv data hookup work done (#25174)

Mottie committed:
DR | Set up shared area of disagreement code (#25157)

* Add shared appeals resources

* Add unit tests

* fix failing test

jsimonVA committed:
Vfep 710 (#25113)

* VFEP-710 #comment handled lint errors FilterBeforeResults.jsx

* VFEP-710 #comment removed unuse keyHandler function FilterBeforeResults.jsx

* #VFEP-710 made changes to scroll effect in NameSearchResults.jsx

* #VFEP-710 #comment created unit test FilterBeforeResults

* #VFEP-710 #comment merged main into VFEP-710

* Delete q

nickjg231 committed:
Add VaProcessList v1 and v3 components to ds-playground page

ataker commented on src/applications/ds-playground/pages/V1V3Page.jsx:
```diff
@@ -827,6 +828,103 @@ export default function V1V3Page() {
+
+            <div className="vads-l-col--12 small-screen:vads-l-col--6 vads-u-margin--1">
+              <VaProcessList uswds>
+                <li>
```
v3 `VaProcessList` should be using `va-process-list-item` as children, not `li`

ataker replied:
Maybe we need to add a linting error to the `VaProcessList` component so this gets caught easier?

Mottie committed:
Update 526 ITF content & add back link (#25180)

OptionSelect committed:
Fix text with conditions per design (#25178)

radavis committed:
Revert "Support message count on MHV landing page (#24791)" (#25170)

This reverts commit a56a62343e5fd3e9e9c08e62f6edd8b6aca28e88.

dellerbie committed:
My VA: Update GA api-name for benefits claims (#25172)

vbahinwillit committed:
61301 Cerner Re-direct on the Appointments homepage (#25160)

* fix to open link in new tab
remove feature flag

* remove unused selector using selectIsCernerPatient

* fix failing unit test

allisonlu committed:
My VA - update error va-alerts to warning variation (#25127)

* add feature toggle on frontend

* - updated errors to warning alerts for claims and
payment history
- put them behind a Toggler

* - updated Appointments API failure error to use
warning variation
- also updated the Debts and Copays API failure errors
- fixed some logic for Outstanding debts (duplicates of the errors were showing)

* - fixed logic for displaying Health care CTA links
on left vs right side
- updated data-test id for Health care error
- updated unit tests for HealthCareContentV2

* set feature flag to true in test

* - update Notification dismissal error to use
va-alert web component instead of the React binding
- also put warning update behind Toggler

* - replaced use of Toggler component with
useFeatureToggle hook instead (ty Adam!)

* cleanup

* - clean up propTypes and removed unused ones

amponce committed:
streamlined waiver submission tracking (#25171)

* streamlined waiver submission tracking

* update submit date tests for streamline

* change order of items to be alphabetical

* Refactor submitForm

josephrlee committed:
[SW] Added external link icon to privacy policy link in checkbox label (#25187)

Mottie committed:
Fix margin between ITF navigation link & button (#25185)

josephrlee committed:
[SW] Changed alerts to standard alerts and update copy on streamlined waiver explainer (#25176)

* Changed Resolution and Household explainer alert to normal style.

* Fixed input width issue for Cash on Hand question.

* Updated StreamlinedExplainer widget alert to standard style and fixed styles and updated copy.

* Updated verbaige.

Commit:
MBMS-45417 Self-Service: Pre-Need: Contact Information Step: Preparer state field should be required for US (#25163)

* MBMS-45417 Moved conditional to ensure correct required fields

* MBMS-45417 added prod flag

chriskim2311 committed:
VACMS-12042: Update props to correct data type (#25098)

VACMS-12042 Update prop type and fix styles for discharge upgrade wizard 

---------

Co-authored-by: Chris Kim <chriskim@Chriss-MacBook-Pro.local>

ediiotero committed:
add error handling for 403 and fixed form config (#25165)

wullaski committed:
Add feature flag to names (#25151)

* Add feature flag to names

* Added unified landing page selector, mocks, and test

* Update the readme with feature toggle stuff

* Fix test

jpresley23-tw committed:
Dhp 937 fix broken tests (#25175)

* [DHP-937] rename dhp-connected-devices so that the tests run on CI

* [DHP-937] fix module path to import  renderInReduxProvider in DHP tests

* [DHP-937] partial fix of DHP TermsAndConditions tests

* DHP-937 Fix DHP TermAndConditions test

Co-authored-by: Mariah Sager <mariah.sager@thoughtworks.com>

* [DHP-937] ensure TermsAndConditions test for dhp worked properly

* [DHP-937] Fix DHP DeviceDisconnectionCard test

Co-authored-by: Oliver Moreno <omorenop@thoughtworks.com>

* [DHP-937] fix text to find for DHP TermsAndConditions test
Co-authored-by: Yoni Doron-Peters <jonathan.d@thoughtworks.com>

---------

Co-authored-by: Mariah Sager <mariah.sager@thoughtworks.com>
Co-authored-by: Oliver Moreno <omorenop@thoughtworks.com>

chriskim2311 committed:
VACMS-14613: Remove scroll from Income Limits (#25107)

VACMS-14613: Remove auto-scroll from income limits pages

---------

Co-authored-by: Chris Kim <chriskim@Chriss-MacBook-Pro.local>

Wyzardsleeves committed:
[MHV-44367] Updated the grey bars in Vitals (#25071)

* [MHV-44367] Updated the grey bars in Vitals

* added maximum width to pagination section

* added date and removed artifacts from pdf download in Care Summary

* updated pagination after meeting UCD

---------

Co-authored-by: mattwrightva <107576133+mattwrightva@users.noreply.github.com>
Co-authored-by: Adrian Rollett <acrollet@gmail.com>

OptionSelect committed:
63393 - [SW] Midpoint Review - IA links and typo updates (63343) (#25184)

* Add exit link per IA review

* Fix transition page test

* Fix long test

robertroddev committed:
Mhv 46752 hi fi mock updates (#25193)

* [MHV-46752] removed grey background, made the name of the rx a link to the details page, and removed the medications history and details part.

* [MHV-46752] renamed Breadcrumbs componend from MrBreadcrumbs to RxBreadcrumbs.

* [MHV-46752] added 40px margin at the bottom of page to seperate from VA footer.

* [MHV-46752] added sort button, added breadcrumbs to top of list page, fixed spelling error of text under h1, and removed resource related to medications section.

* [MHV-46752] realigned sort to be next to sorting list.

* [MHV-46752] added dispatch to breadcrumb useEffect hook.

Thrillberg committed:
21-0972 content updates (#25058)

* WIP - Start work on 21-0972 content updates

* WIP - continuing down the form...

* Everything but the confirmation page

* first pass done

* updates per feedback

* fix tests and remove unnecessary line

* e2e tests

* Fix schemas per feedback

* Some more tweaks

* fix silly mistake

fazilqa committed:
Mhv 47274 move message by keyboard (#25186)

* add move-message-by-keyboard spec

* add realEvents command

* add tests to existing spec

* update the selectors

* update the methods

* remove unnecessary checks

* upd methods

* upd methods

* upd intercept for signature API call

* skip unread.message.count.cypress.spec.js

nickjg231 committed:
Merge with origin/main

nickjg231 committed:
Update V3 VaProcessList to use <VaProcessListItem> instead of <li>
